src/usr/local/www/services_dyndns_edit.php

```diff
@@ -491,6 +492,9 @@ function setVisible(service) {
 				hideInput('ttl', false);
 				break;
 			case "namecheap":
+			case "cloudflare-v6":
+			case "cloudflare":
```

Comment:
I suspect something like https://github.com/pfsense/pfsense/pull/3717 is what is needed to integrate the 2 changes that have caused this. But I don't have Cloudflare, so @CarlGill or someone will need to confirm.
The change should allow the "proxied" field to appear in the cloudflare case.